### PR TITLE
Desktop version page

### PR DESCRIPTION
**Pull request by Ejikeokegbe**
Added some Figma templates for more portfolio description
Added some features to enable desktop display on mobile first website
This shows the power of media queries on making a website responsive

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -89,43 +93,44 @@ body {
...
-.items {
+.content {
   padding-left: 20px;
   margin-right: 50px;
 }
 
-.items h1 {
+.content h1 {
   color: #172b4d;
+  font-family: Poppins, sans-serif;
```
Comment (Shisui6) [inline style.css:130]: Currently, your fonts are not being applied in the browser. Please make sure that you are importing and using the correct font family used in the figma template then update **all** your text elements to use the correct font family🙏

File: style.css [hunk 2]
```diff
@@ -264,16 +270,230 @@ body {
...
+
+  .container-b {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 2%;
+    padding: 1rem 0;
+    border-radius: 24px;
+  }
```
Comment (Shisui6) [inline style.css]: The styles you applied to your desktop version are rearranging the content in the works section of your mobile version, as seen in the screenshots below:

Your design:
image

Figma design:
image

Please make your that your mobile version is unaffected by your desktop styling🙏

File: style.css [hunk 3]
```diff
@@ -264,16 +270,230 @@ body {
...
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
```
Comment (Shisui6) [inline style.css]: Please resize the background image on your headline section to match the figma design better. Currently, the image is stretched.

Your design:
image

Figma design:
image

File: style.css [hunk 4]
```diff
@@ -264,16 +270,230 @@ body {
...
+
+  .desktop {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-content: center;
+    background-color: #f7f7f9;
+    padding-bottom: 5%;
+  }
```
Comment (Shisui6) [inline style.css:379]: Kindly update your works section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Please reduce the left and right margins on each card
- Please reduce the left and right padding on each card
- Please increase the size of the image to match the figma design
- Please reduce the height of the text content so it doesn't span the full height of the card and matches the design

File: style.css [hunk 5]
```diff
@@ -264,16 +270,230 @@ body {
...
+
+  .container-c {
+    display: grid;
+    grid-template-columns: 1.4fr 2fr;
+  }
+
+  .about-div {
+    padding-left: 50px;
+  }
```
Comment (Shisui6) [inline style.css:430]: Kindly update your about section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Please increase the spacing between the elements on the left-hand side
- Please update the color of the `let's connect` heading
- Please increase the spacing between the technologies (HTML, CSS, JS)

File: style.css [hunk 6]
```diff
@@ -264,16 +270,230 @@ body {
...
+  background-size: contain;
+  background-position: right;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 100px 0 0 0;
+  padding: 4rem 1rem;
+  gap: 3rem;
+  margin-top: 1rem;
+}
```
Comment (Shisui6) [inline style.css]: Kindly update the styling of your contact section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Please resize the background image so it matches the figma design
- Please remove the border radius on the input fields on the desktop version

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Ejikeokegbe, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Good use of media queries ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

- Please remove the FAQ section from your readme file since it is not being used. This will make your readme look more professional

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 7]
```diff
@@ -46,24 +51,25 @@ body {
...
+  line-height: 30px;
+  padding-top: 80px;
 }
 
 .leading {
   font-size: 18px;
   text-align: start;
-  margin: 20px 0;
+  margin: 10px 0;
 }
```
Comment (tresorsawasawa) [inline style.css]: - [ ] As mentioned by the previous reviewer and for the project requirements to match your Headline section design to the one from Figma design, kindly ensure that you use the correct `text-color` on your heading and leading text. Here are the correct colors to be used: `#091e42` on the `Heading` and `#344563` on the `leadding` text. 
**Make sure you apply this on both Mobile and Desktop**

File: index.html [hunk 8]
```diff
@@ -124,14 +138,107 @@ <h1>Multi-Post Stories</h1>
...
+            <li><a href="#" class="btn">javascript</a></li>
+          </ul>
+          <button class="last-btn" type="button">See Project</button>
+        </div>
+        <img src="pimages/Snapshoot Portfolio (1).png" />
+      </div>
+    </section>
     <section class="section-c">
       <div class="container-c">
         <div class="about-div">
```
Comment (tresorsawasawa) [inline index.html:244]: - [ ] As mentioned by the previous reviewer to stick as much as possible to Figma design, I think there are things should improve again such as the `fonts(size, weight,...)`, `color` on some text, `spacing`(padding or margin), `alignment`,...
Just take a look on the difference on the above 2 images:
### Figma Design 
Screenshot from 2023-05-12 17-57-57

### Your Design 
Screenshot from 2023-05-12 18-17-39

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Ejikeokegbe 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Ejikeokegbe 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Ejikeokegbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x] Implemented changes requested by the previous reviewer :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark:   

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**